modules/oe_content_event/config/install/core.entity_form_display.node.oe_event.default.yml

```diff
@@ -403,13 +403,17 @@ content:
     settings:
       form_mode: default
       revision: true
-      label_singular: ''
-      label_plural: ''
-      override_labels: false
-      collapsible: false
+      override_labels: true
+      label_singular: venue
+      label_plural: venues
+      collapsible: true
+      allow_new: true
+      match_operator: CONTAINS
       collapsed: false
+      allow_existing: false
+      allow_duplicate: false
     third_party_settings: {  }
-    type: inline_entity_form_simple
+    type: inline_entity_form_complex
```

Comment:
As I understood we should use complex widget for any entity references

Reply:
It is not the case with Venue. Wasn't part of the ticket so please restore it as it was because that was. In general, you should not change existing code based on assumptions and without clear request in the ticket, because you will likely cause regression and we have no documentation over why the change has been implemented.

Reply:
@ademarco @drishu As I see there were discussion about it. Please clarify what widget we should to use here.

Reply:
Yeah let's not change this right now.